packages/dashboard/src/components/DashboardFrame/index.tsx

```diff
@@ -22,44 +87,89 @@ const useStyles = makeStyles((theme) => ({
-                    <Navigation />
-                </Grid>
-                <Grid container direction="column" item xs={10}>
+                {!matches && (
+                    <Grid item xs={2} className={classes.permanentDrawer}>
+                        <Navigation />
+                    </Grid>
+                )}
+                <Grid container direction="column" item xs={matches ? 12 : 10} className={classes.rightContainer}>
                     <ErrorBoundary>{props.children}</ErrorBoundary>
                 </Grid>
             </Grid>
-        </>
+        </DashboardContext.Provider>
     )
 }
 
 export interface PageFrameProps extends React.PropsWithChildren<{}> {
     title: React.ReactNode | string
     primaryAction?: React.ReactNode
 }
+
 export function PageFrame(props: PageFrameProps) {
     const classes = useStyles()
     const left = typeof props.title === 'string' ? <Typography variant="h6">{props.title}</Typography> : props.title
     const right = props.primaryAction
+    const matches = useMediaQuery<Theme>((theme) => theme.breakpoints.down('lg'))
+    const { drawerOpen, toggleDrawer } = useContext(DashboardContext)
     return (
         <>
             <AppBar position="relative" color="inherit" elevation={0}>
-                <Toolbar component={Grid} container>
-                    {left}
-                    <Box sx={{ flex: 1 }} />
-                    {right}
+                <Toolbar component={Grid} container className={classes.toolbar}>
+                    {matches && (
+                        <Grid item xs={2} container alignItems="center" className={classes.logo}>
+                            <IconButton onClick={toggleDrawer} className={classes.menuButton}>
+                                {drawerOpen ? <CloseIcon /> : <MenuIcon />}
+                            </IconButton>
+
+                            <img height={40} alt="Mask Logo" src="https://mask.io/assets/icons/logo.svg" />
+                        </Grid>
+                    )}
+                    <Grid item xs={matches ? 10 : 12} container className={classes.title}>
+                        {left}
+                        <Box sx={{ flex: 1 }} />
+                        {right}
+                    </Grid>
                 </Toolbar>
             </AppBar>
             <Grid item xs className={classes.containment}>
+                {matches && (
+                    <Drawer
+                        open={drawerOpen}
+                        onClose={toggleDrawer}
+                        BackdropProps={{ invisible: true }}
+                        PaperProps={{ elevation: 0 }}
+                        variant="temporary"
+                        ModalProps={{
+                            BackdropProps: {
+                                classes: { root: classes.drawerBackdropRoot },
+                            },
+                        }}
+                        classes={{ paper: classes.temporaryPaper, root: classes.drawerRoot }}>
```

Comment:
Notice when you're changing a [deeper element](https://next.material-ui.com/guides/interoperability/#deeper-elements-3), the default className values like `.MuiSlider-thumb` does not working in our project. You may keep it for now too. I still not figure out what's the best practise for this style.